kahuna/public/js/components/gu-date/gu-date-range-x.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <gu-date class="full-width" label="from" date="start" max-date="end"></gu-date>
```

Comment:
is the max date of start the end? Getting confused now, let's look at this on Monday please :)

Reply:
> is the max date of start the end?

Yes. Possibly not very intention revealing though. The idea here is in a date range, the start date should not exceed the end date, else its an invalid range.